src/plugins/ability.js

```diff
@@ -55,6 +55,11 @@ export default ({ store }, inject) => {
   inject('canPatch', (name, obj) =>
     store.getters.isAbilityUpdated ? ability.can('patch', as(name, obj)) : false
   )
+  inject('canRemove', (name, obj) =>
```

Comment:
did Scott resolve the issues in the api, for this ability?

Reply:
Not yet, I am planning to delete this ability

Reply:
sounds good